src/main/java/uk/gov/pay/ledger/report/params/TransactionSummaryParams.java

```diff
@@ -68,12 +81,27 @@ public void setToDate(String toDate) {
         return filters;
     }
 
+    public List<String> getFilterTemplatesWithMoto() {
+        List<String> filters = getFilterTemplates();
+
+        if (isMoto()) {
+            filters.add(" t.moto = :" + MOTO);
+        }
+
+        return filters;
+    }
+
     public Map<String, Object> getQueryMap() {
         Map<String, Object> queryMap = new HashMap<>();
 
         if (isNotBlank(accountId)) {
             queryMap.put(GATEWAY_ACCOUNT_EXTERNAL_FIELD, accountId);
         }
+
+        if (moto) {
+            queryMap.put(MOTO, true);
+        }
```

Comment:
I think this is getting a bit confused between the query parameter indicating whether or not we should include statistics about the moto payments and whether payments are moto or not. Might be easier to talk it through.